### ISSUE TITLE
Ambiguous reference 'Options' could either refer to 'Graphics.Blank.Options' or 'Data.aeson.options'

### ISSUE THREAD

**Issue by aadi123** [ku-fpg/blank-canvas#88]
Tried installing blank canvas using the following command on an ubuntu 16.04 machine
`cabal install blank-canvas`

and I get the following error message
```
Graphics/Blank.hs:233:16:
    Ambiguous occurrence ‘Options’
    It could refer to either ‘Graphics.Blank.Options’,
                             defined at Graphics/Blank.hs:410:1
                          or ‘Data.Aeson.Options’,
                             imported from ‘Data.Aeson’ at Graphics/Blank.hs:173:1-27
                             (and originally defined in ‘aeson-1.2.2.0:Data.Aeson.Types.Internal’)

Graphics/Blank.hs:419:14:
    Ambiguous occurrence ‘Options’
    It could refer to either ‘Graphics.Blank.Options’,
                             defined at Graphics/Blank.hs:410:1
                          or ‘Data.Aeson.Options’,
                             imported from ‘Data.Aeson’ at Graphics/Blank.hs:173:1-27
                             (and originally defined in ‘aeson-1.2.2.0:Data.Aeson.Types.Internal’)
```

**Comment by RyanGlScott** [comment]
Ugh. This is due to a change in `aeson-1.2.2.0` where `aeson`'s `Options` are now exported from `Data.Aeson` (as opposed to before, where it was sequestered off in `Data.Aeson.TH`). Alas, this conflicts with `blank-canvas`'s own `Options`. For the time being, I've restricted `blank-canvas`'s upper version bounds on `aeson` to exclude `aeson-1.2.2.0` on Hackage.

I've fixed this in `blank-canvas` HEAD (see 02e4cb070603e1d6a2f36c8a482df6fcbcc7b410), but there remains a question of how we should release a new version with this change. Alas, the version of `blank-canvas` on Hackage (0.6) is far out of sync from the version in this repo (0.6.1), and the version in this repo is in no shape to be released right now (see #87).

One option is to backport 02e4cb070603e1d6a2f36c8a482df6fcbcc7b410 back to 0.6 and release an emergency fix as `blank-canvas-0.6.1` (and bump the version in the repo ahead of that to, say, at least 0.6.2).
